src/components/ProfilePicPopup.js

```diff
@@ -0,0 +1,83 @@
+import Popup from "./Popup";
+
+class ProfilePicPopup extends Popup {
```

Comment:
Please, delete `ProfilePicPopup` because you need to use `PopupWithForm` for any modal with a form (including the avatar modal)

Reply:
Done. Indeed, it could be done only with the use of popup with form